src/components/Trip.jsx

```diff
@@ -1,16 +1,25 @@
 import React, { useState } from "react";
 import { connect } from "react-redux";
 import { Dropdown, Grid, Button } from "semantic-ui-react";
-import { addActivityType, addHotels } from "../modules/destination.js";
+import {
+  addActivityType,
+  addHotels,
+  addRestaurants
+} from "../modules/destination.js";
 
 const Activities = props => {
```

Comment:
This component has gotten very big. Consider splitting into multiple sub-components.

Reply:
Good point. Refactored it.